oe_webtools.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Webtools Integration
+description: OpenEuropa webtools integration.
+package: OpenEuropa Webtools
```

Comment:
Are we sure about the package here ? Shouldn't it be `OpenEuropa` only ?

Reply:
No requirement for that and even recommendation, so it is open for discussion!